app/models/plant_population.rb

```diff
@@ -87,9 +87,7 @@ def as_indexed_json(options = {})
   def self.permitted_params
     [
       :fetch,
-      query: [
-        :id, :name
-      ]
+      query: [:id, :name]
```

Comment:
I will revert that so we don't get conflicts. Anyway, it's only formatting change.